lib/Document/Adapter/Ghostscript.php

```diff
@@ -185,7 +185,7 @@ protected function buildPageCountCommand()
     protected function getVersion()
     {
         if (is_null($this->version)) {
-            $this->version = (float) Console::exec(self::getGhostscriptCli() . ' --version');
+            $this->version = Console::exec(self::getGhostscriptCli() . ' --version');
```

Comment:
PHP doc type of the atttribute should be also changed, right?

Reply:
Fixed, thanks.

Reply:
I mean not the method return (but is also needed, yes) but the attribute var doc